examples/training/classification/densenet/imagenet/basic_training.py

```diff
@@ -123,13 +125,14 @@ def load_imagenet_dataset():
 
 
 """
-Next, we augment our dataset. We define a set of augmentation layers and then apply them to our input dataset using the `apply_augmentation` method from our KerasCV training utils.
+Next, we augment our dataset.
+We define a set of augmentation layers and then apply them to our input dataset.
 """
 
 AUGMENT_LAYERS = [
     keras_cv.layers.RandomFlip(),
     keras_cv.layers.RandAugment(value_range=(0, 255), magnitude=0.3),
```

Comment:
Can be added in a future PR, but:

+RandomlyResizedCrop(target_size=(224, 224), zoom_factor=(0.8, 1.0), aspect_ratio_factor=(3/4, 4/3))